### PR TITLE
Allow setting of countryCode externally.

### PR DESCRIPTION
**Pull request by tkn**
By using the countryCode var in both strings, setting the country code externally will work.
E.g.
`var tidal = new Tidal();
tidal.countryCode = "DK";`

### CODE REVIEW

**Comment by tkn** [comment]
I don't understand why the CI build failed with tests not passing. The default countryCode shouldn't have changed as far as I can see.

**Comment by tkn** [comment]
This doesn't seem to work at all. My javascript skills are lacking, but it would be useful to be able to set the country code, since results differ by country,

**Comment by spencercharest** [comment]
Hey @tkn I appreciate the contribution. I agree that this would be a great feature to add. I think I'll add an option to define the country code and API limit when the class is instantiated. By the way, the build was failing due to an error in my CI configuration. The tests rely on some environment variables which it looks like were only accessible to myself as a user. This is the first time anyone has contributed so I hadn't noticed before. I'll have to get that sorted out as well.

**Comment by tkn** [comment]
That would be great. Apologies for the lousy contribution - I am a total novice at JavaScript and node.js,

I'm using your library to hack together a script that searches for tracks in my Tidal account that are available in a higher quality. For some reason Tidal Masters are separate from the "regular" lossless tracks, so it's possible to add e.g. "Fortunate Son" in the regular lossless version rather than the (arguably) higher quality MQA version - and for some reason Tidal doesn't have a way to detect this automatically.

Anyway, long story short, the availability on Tidal varies by country, and I'm in Denmark, so I needed to change the country code, but other than that, your wrapper has been fantastic.

**Comment by spencercharest** [comment]
@tkn it's great to hear that it's useful for someone. I put this together for a separate project I was working on but figured I'd publish it in case anyone else could use it. I've pushed an update to NPM. The class instantiation now takes an options object where `countryCode` and `limit` can be explicitly set. Take a look at the new documentation here: 

https://github.com/spencercharest/tidal-api/blob/master/docs/api.md#new_Tidal_new

**Comment by tkn** [comment]
Works great. Thanks!
